### DISCUSSION TITLE
What regex "flavor" does Noptepadqq use?

### DISCUSSION THREAD

**Discussion by CalvinPryor** [notepadqq/notepadqq#1042] [Q&A] [answered]
I have a couple of regular expressions that work great for cleaning up dates in Noptepadqq. 
But I'm having a tough go at automating this using these same regular expressions with sed. 
Trying to rule out "flavor" conflict.

**Comment by rsaxvc** [comment, marked as answer]
It's QRegularExpression, which is mostly PCRE compatible.

See also https://stackoverflow.com/a/70223237/1125660
